### PR TITLE
add firebase credential to Dockerfile

### PR DESCRIPTION
**Pull request by Fenydude**

### CODE REVIEW

**Review by OlgaEvdokimova — DISMISSED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -13,5 +14,6 @@ FROM openjdk:11-jdk-slim
 ARG path=/usr/app
 WORKDIR ${path}
 COPY --from=build ${path}/target/*.jar ${path}/app.jar
+COPY --from=build ${path}/*.json ${path}/internship-cloud/internship-project.json
```
Comment (SergeyKoval) [inline Dockerfile]: Why are you coping all json files if you need just one?

Comment (Fenydude) [reply Dockerfile]: I did this because we only have one json file. fixed

File: Dockerfile [hunk 2]
```diff
@@ -5,6 +5,7 @@ FROM maven:3.6.2-jdk-11-slim AS build
 WORKDIR /usr/app
 COPY src ./src
 COPY pom.xml .
+COPY internship-cloud/*.json ./internship-project.json
```
Comment (SergeyKoval) [inline Dockerfile]: Why are you coping it to the build container first? Copy it only to the final container

**Review by SergeyKoval — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (Fenydude) [reply Dockerfile]: fixed

File: Dockerfile [hunk 3]
```diff
@@ -13,5 +13,6 @@ FROM openjdk:11-jdk-slim
 ARG path=/usr/app
 WORKDIR ${path}
 COPY --from=build ${path}/target/*.jar ${path}/app.jar
+COPY internship-cloud/*.json ${path}/internship-cloud/internship-project.json
```
Comment (SergeyKoval) [inline Dockerfile]: why again all json files?

File: src/main/java/by/exadel/internship/service/impl/InterviewServiceImpl.java [hunk 4]
```diff
@@ -99,9 +98,9 @@ public void saveInterviewForForm(UUID formId, InterviewSaveDTO interviewDTO) {
             formFullDTO.setInterview(interviewFullDTO);
             formFullDTO.setFormStatus(FormStatus.ADMIN_INTERVIEW_ASSIGNED);
 
-            emailService.sendInterviewDateOnEmail(formFullDTO.getEmail(),
+            /*emailService.sendInterviewDateOnEmail(formFullDTO.getEmail(),
```
Comment (SergeyKoval) [inline src/main/java/by/exadel/internship/service/impl/InterviewServiceImpl.java]: remove code if you don't need it

Comment (Fenydude) [reply src/main/java/by/exadel/internship/service/impl/InterviewServiceImpl.java]: Fixed

File: Dockerfile (hunk 3 above)
Comment (Fenydude) [reply Dockerfile]: Fixed. I want to keep "*.json" because now we have to constantly change the file name to the one we need

**Review by SergeyKoval — APPROVED** [review]

**Review by Maksim-maevsky — APPROVED** [review]

**Pull request merged**